osgi.enroute.web.simple.provider/src/osgi/enroute/web/server/provider/DispatchServlet.java

```diff
@@ -70,7 +70,7 @@ public void service(HttpServletRequest rq, HttpServletResponse rsp) throws Servl
 
 		// No ConditionalServlets were found. Since we don't know what to do, we
 		// return a 404.
```

Comment:
Unless we actually send the error at some point, we don't see the 404 page. Perhaps this is not the best place to do it, but this seems to work.